.travis.yml

```diff
@@ -4,7 +4,7 @@ os:
 language: go
 
 go:
-  - 1.11.x
+  - 1.12.x
```

Comment:
This is needed because one of the go-datastore tests from a version post v0.0.1 uses a function added in go 1.12.

Should we bump this all the way to go 1.13?

Reply:
Let's stick with the minimum supported version.